File: swagger/models/get_client_formula_notes_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse
from .formula_note import FormulaNote

class GetClientFormulaNotesResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		formula_notes
		Contains details about the client’s formula.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'FormulaNotes': ['formula_notes', FormulaNote],
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'formula_notes': 'FormulaNotes',
		}

	pagination_response = None
	formula_notes = None

File: swagger/Models/GetClientFormulaNotesResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetClientFormulaNotesResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains details about the client’s formula.
	 * 
	 * @var FormulaNote[] 
	 */
	public $formulaNotes;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'FormulaNotes' => ['formulaNotes', 'array', FormulaNote::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'formulaNotes' => ['FormulaNotes', 'array', FormulaNote::class],
			];
	}
}
